lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -322,6 +388,92 @@ public void OnError(Exception error)
             }
         }
 
+        private sealed class BreakTestTask : ITask, IObserver<ICloseEvent>
```

Comment:
I do not see why this should be problem for other reviewers if names are better..Anycase.